src/commands/create.rs

```diff
@@ -26,8 +29,23 @@ pub fn create() -> Result<()> {
     }
 
     if !working_area.has_staged_changes() {
-        // TODO: Add a prompt to ask the user if they want to stage all changes
-        git_client.add_all()?;
+        let items = vec!["Commit all changes (--all)", "Abort operation"];
```

Comment:
I imagine that in the future we'll add something like "Select files to add"

Reply:
Yes, add with `--patch` option. https://nuclearsquid.com/writings/git-add/